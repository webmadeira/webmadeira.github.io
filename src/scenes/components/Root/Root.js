import styled from 'styled-components'

const Root = styled.div`
  display: flex;
  flex-direction: column;
  height: 100%;
`

export default Root
