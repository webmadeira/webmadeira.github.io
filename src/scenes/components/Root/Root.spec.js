import React from 'react'
import { shallow } from 'enzyme'
import Root from './Root'

describe('Home\'s scene Root component', () => {
  it('must render properly', () => {
    const component = shallow(<Root>Content</Root>)

    expect(component).toMatchSnapshot()
  })
})
