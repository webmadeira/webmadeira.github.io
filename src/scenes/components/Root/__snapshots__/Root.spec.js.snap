// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home's scene Root component must render properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <styled.div>
    Content
  </styled.div>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Content",
      "className": "sc-bdVaJa jaytjN",
    },
    "ref": null,
    "rendered": "Content",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Content",
        "className": "sc-bdVaJa jaytjN",
      },
      "ref": null,
      "rendered": "Content",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
